importService/src/importFileParser.js

```diff
@@ -0,0 +1,60 @@
+import AWS from 'aws-sdk';
+import csv from 'csv-parser';
+
+const s3 = new AWS.S3();
+AWS.config.update({ region: 'eu-west-1' });
+
+const filePrefix = process.env.IMPORT_FILE_PREFIX;
+const copyPrefix = process.env.COPY_PREFIX;
+
+const copyFileToProcessedFolder = (bucket, fileName) => {
+  const destination = fileName.replace(filePrefix, copyPrefix);
+  const params = {
+    Bucket: bucket,
+    CopySource: `${bucket}/${fileName}`,
+    Key: destination,
+  };
+  console.log('Starting file copy with params', params);
+  return new Promise((resolve, reject) => {
+    s3.copyObject(params, function (err, data) {
+      if (err) reject(err);
+      else resolve(data);
+    });
+  });
+};
+
+const deleteObject = async (bucket, key) => {
+  var params = {
+    Bucket: bucket,
+    Key: key,
+  };
+
+  console.log('Starting file deletion with params', params);
+  return new Promise((resolve, reject) => {
+    s3.deleteObject(params, (err, data) => {
```

Comment:
just suggestion, there is promisifyied methods: s3.deleteObject(params).promise()